### ISSUE TITLE
Models using CMNetworkEntity ?

### ISSUE THREAD

**Issue by SergeStinckwich** [cormas/cormas#462]
Do we have a model using ``CMNetworkEntity`` ? Apparently this is a class to represent oriented graph made with nodes (CMNodeEntity) and arcs (edges) (CMArcEntity).

``CMNodeEntity``, ``CMArcEntity`` might be renamed more simply as ``CMNode`` and ``CMArc``.

**Comment by pierbo** [comment]
Yes, there are several models using those entities (Arc & Node). In this case, an instance of NetworkEntity is automatically created as an oriented network : rivers, roads, etc. By default, it is oriented. We should generalize it in order to get other types of networks.
Yes we can rename as 'CMNode' and 'CMArc'

**Comment by SergeStinckwich** [comment]
At the moment, we have no models that used CMNetWorkEntity. Which models use this?

**Comment by pierbo** [comment]
On Cormas-VW website, there is a page with several models: [http://cormas.cirad.fr/en/applica/tousmodeles.htm](http://cormas.cirad.fr/en/applica/tousmodeles.htm) (and not all of them are listed). 
Some of them use networks (see for ex "ContaMiCuenca" or "Demo_Aquifer").
In the futur, the majority of those models should run on Cormas-Pharo (with some manual modifications, as we agreed).
